azure-functions/acbs-function/package.json

```diff
@@ -25,7 +25,8 @@
     "prettier": "prettier --no-error-on-unmatched-pattern --check **/*.{ts,js} --ignore-path ../../.prettierignore",
     "prettier:fix": "prettier --write **/*.{ts,js} --ignore-path ../../.prettierignore",
     "unit-test": "jest --coverage --verbose --config=unit.jest.config.js",
-    "unit-test-dev": "jest --verbose --config=unit.jest.config.js"
+    "unit-test-dev": "jest --verbose --config=unit.jest.config.js",
+    "unit-test-ff": "jest --coverage --verbose --config=unit.ff.jest.config.js --passWithNoTests"
```

Comment:
What's the reason for adding this please, especially with the `passWithNoTests` flag?

Looks like this isn't consumed anywhere

Reply:
This is to ensure feature flag unit test cases can be executed and there can be occasions when no unit tests for feature flags exists. I have added this across all the micro-services.

Reply:
Thanks @abhi-markan makes sense - should we add this to the CI/CD pipeline? If so I would suggest in a new PR

Reply:
Most definitely not for this PR but I'm quite strongly against having config to run unit tests in particular with feature flags. Unit tests test the actual implementation on the lowest level and should be aware of feature flags, so our tests should be in charge of mocking what the value of certain env variables/feature flags are rather than the CI server. This comment definitely doesn't apply to other tests like E2E tests which are abstracted away where I do think the CI/CD pipeline should be in charge of setting the variables.

Ignore this comment for this PR as not for now but let's definitely organise that chat re feature flag testing especially as we're looking to bring more in soon